data_summaries/DialogStudio.json

```diff
@@ -0,0 +1,3826 @@
+{
+  "ds-ABCD": {
+    "Unique Dataset Identifier": "ds-ABCD",
+    "Dataset Name": "ABCD",
+    "Paper Title": "Action-Based Conversations Dataset: A Corpus for Building More In-Depth Task-Oriented Dialogue Systems",
+    "Dataset URL": "https://github.com/asappresearch/abcd",
+    "GitHub URL": "https://github.com/asappresearch/abcd",
+    "Hugging Face URL": "",
+    "Papers with Code URL": "https://paperswithcode.com/dataset/abcd",
+    "ArXiv URL": "https://aclanthology.org/2021.naacl-main.239",
+    "Semantic Scholar Corpus ID": "",
+    "Collection": "DialogStudio",
+    "Collection URL": "https://github.com/salesforce/DialogStudio",
+    "Languages": [
+      "'English'"
+    ],
+    "Task Categories": [
+      "'Context Generation'",
+      "'Dialogue Generation'",
+      "'Dialog Turn Prediction'",
+      "'Dialogue State Tracking'",
+      "'Dialogue Act Recognition'"
+    ],
+    "Text Sources": [
+      "'human'"
+    ],
+    "Model Generated": [
+      ""
+    ],
+    "Format": [
+      "'Multi-turn Dialog'"
+    ],
+    "Human Annotation": "Yes",
+    "Derived from Datasets": [
+      ""
+    ],
+    "Creators": [
+      ""
+    ],
+    "License": "MIT License",
+    "License URL": "https://github.com/asappresearch/abcd/blob/master/LICENSE",
+    "License Verified By": "Kun Qian",
+    "Licenses": {
+      "License": "MIT License",
+      "License URL": "https://github.com/asappresearch/abcd/blob/master/LICENSE"
+    },
+    "Dataset Filter IDs": ""
```

Comment:
This field can be used as a key to the dataset that you can pass directly into the huggingface dataset load function, or to filter for just examples from this dataset.